apps/dataset-ingestion/app/app.sh

```diff
@@ -30,31 +30,32 @@ fi
 # gcloud uses it's bundled python. Do not inject our sitecustomize.
 PYTHONPATH='' gcloud config set auth/disable_credentials True
 
+update_phases() {
+    args=("--phases" "val_downloading" "--phases" "val_ingesting_images" "--phases" "val_ingesting_bounding_boxes" "--phases" "val_ingesting_images" "--phases" "val_ingesting_connections" "--phases" "val_ingesting_polygons" "--phases" "val_ingesting_descriptors" "--phases" "val_ingesting_connections")
+    if [[ $INCLUDE_TRAIN == true ]]; then
+        args+=("--phases" "train_downloading" "--phases" "train_ingesting_images" "--phases" "train_ingesting_bounding_boxes" "--phases" "train_ingesting_images" "--phases" "train_ingesting_connections" "--phases" "train_ingesting_polygons" "--phases" "train_ingesting_descriptors" "--phases" "train_ingesting_connections")
+    fi
+    python3 $STATUS_SCRIPT --completed 0 "${args[@]}"
+}
+
```

Comment:
I feel like having each phase on a separate line is more readable, can you do it with this too?

Reply:
sure thing . will do.